src/utils/index.ts

```diff
@@ -17,24 +17,37 @@ export const phoneNumLengthRegEx = (length: number) => {
   return new RegExp(`^\\d{${length}}$`);
 };
 
+export const getPalleteMode = (mode: ThemeMode | null): PaletteMode => {
+  if (mode === ThemeMode.dark) return 'dark';
+
+  return 'light';
+};
+
 export const getValueFromLocalStorage = <T>(key: string): T | null => {
   try {
-    const value = localStorage.getItem(key);
+    const serializedData = localStorage.getItem(key);
+    if (serializedData === null) return null;
 
-    if (value !== null) {
-      return JSON.parse(value) as T;
+    try {
+      return JSON.parse(serializedData) as T;
+    } catch {
+      return serializedData as T;
     }
-
-    return null;
   } catch (error) {
     console.error('Error retrieving value from local storage:', error);
 
     return null;
   }
 };
 
-export const getPalleteMode = (mode: ThemeMode | null): PaletteMode => {
-  if (mode === ThemeMode.dark) return 'dark';
-
-  return 'light';
+export const setValueToLocalStorage = (key: string, value: object | string): void => {
+  try {
+    if (typeof value === 'object') {
+      localStorage.setItem(key, JSON.stringify(value));
+    } else {
+      localStorage.setItem(key, value);
+    }
+  } catch (error) {
+    console.error('Error setting data to local storage:', error);
```

Comment:
need really notify user not only in console. As I remember, we have notifications component